runtime/src/iree/builtins/ukernel/arch/arm_64/mmt4d_arm_64_base.c

```diff
@@ -264,3 +264,120 @@ IREE_UK_MMT4D_TILE_FUNC_IMPL_FOR_M0_1_2_4_8(
     iree_uk_mmt4d_tile_s8s8s32_2x8x1_arm_64,
     iree_uk_mmt4d_tile_s8s8s32_4x8x1_arm_64,
     iree_uk_mmt4d_tile_s8s8s32_8x8x1_arm_64)
+
+static inline void iree_uk_mmt4d_tile_s8s4s32_1x16x2_to_4x16x2_arm_64(
+    void* IREE_UK_RESTRICT out_tile, const void* IREE_UK_RESTRICT lhs_panel,
+    const void* IREE_UK_RESTRICT rhs_panel,
+    const iree_uk_mmt4d_params_t* params, int M0) {
+  IREE_UK_ASSERT(M0 >= 1 && M0 <= 4 && iree_uk_is_po2_u32(M0));
+  IREE_UK_ASSERT(!(params->K0 % 2));
+  const iree_uk_int8_t* IREE_UK_RESTRICT lhs_ptr = lhs_panel;
+  const iree_uk_int8_t* IREE_UK_RESTRICT rhs_ptr = rhs_panel;
+
+  iree_uk_int32_t* IREE_UK_RESTRICT out_ptr = out_tile;
+  int32x4_t acc[16];
+  if (params->flags & IREE_UK_FLAG_MMT4D_ACCUMULATE) {
+    for (int i = 0; i < 4 * M0; ++i) {
+      acc[i] = vld1q_s32(out_ptr + 4 * i);
+    }
+  } else {
+    for (int i = 0; i < 4 * M0; ++i) {
+      acc[i] = vdupq_n_s32(0);
+    }
+  }
+  const int8x8_t vmask0f = vmov_n_s8(INT8_C(0x0F));
```

Comment:
You can drop the `INT8_C` macro, it doesn't make a difference here.

Reply:
Removed